concepts/architecture/index.md

```diff
@@ -300,23 +305,21 @@ Committee selection is dynamic and stake-based, with a new committee elected for
 
 The consensus protocol makes use of:
 
-- The communication layer for consensus round messaging and consensus state synchronisation.
-- The Autonity Protocol Contract for committee selection logic.
+- The [communication layer](/concepts/architecture/#communication-layer) for consensus round messaging and consensus state synchronisation in a [dedicated consensus channel](/concepts/system-model/#p2p-networking-protocols).
+- The [Autonity Protocol Contract](/concepts/architecture/#autonity-protocol-contract) for [committee selection logic](/reference/api/aut/op-prot/#computecommittee).
 
-To learn more about the concept, see [consensus](/concepts/consensus/) and the [protocol parameters](/reference/protocol/) reference.
+To learn more about the concept, see [Consensus](/concepts/consensus/), [System model, Networking](/concepts/consensus/) and the [protocol parameters](/reference/protocol/) reference.
 
 ## Communication Layer
 
 Autonity uses a [fully connected network topology](/glossary/#mesh-network) with peer-to-peer communication based on Ethereum [devp2p](https://github.com/ethereum/devp2p) network protocols RLPx and wire protocol.
 
-Each participant maintains a current record of peers in the network, updated as new participants join or leave the system. Participants establish an authenticated connection with one another over TCP by the RLPx transport protocol. At the application-level, Autonity extends the Ethereum wire protocol for message broadcast to:
+At the P2P level, Autonity separates transaction and consensus traffic on to separate channels on different TCP ports.
+
+For transaction gossiping between nodes the Ethereum wire protocol is used for P2P propagation of block announcements and transactions. Reliable broadcast between network peer nodes and duplicate message send prevention under an [eventually synchronous model](/concepts/system-model/#networking) to guarantee the liveness property of consensus messaging in the wire protocol. Each participant maintains a current record of peers in the network, updated as new participants join or leave the system. Participants establish an authenticated connection with one another over TCP by the RLPx transport protocol. 
```

Comment:
The part about reliable broadcast and eventually sync model does not make much sense to me. What is the meaning?

Reply:
Bad wording.

I first reworded this to:

>Reliable broadcast between network peer nodes is given by an _eventually synchronous model_ as described in the Concept [System model, Networking](/concepts/system-model/#networking). 

Then reworded the whole section again to:

>For transaction gossiping between nodes the Ethereum wire protocol is used for P2P propagation of block announcements and transactions. Each participant maintains a current record of peers in the network, updated as new participants join or leave the system. Participants establish an authenticated connection with one another over TCP by the RLPx transport protocol. For detail, see the Concept description [System model, Networking](/concepts/system-model/#networking). 

That removes text redundancy between the pages.

Please see commit 04e31cf9f4e3471213336bb655d0b265f6e674d2.